Rework Style

Starkium opened the pull request:
Particles demo was causing issues for people, reworked this to be more compatible.

There's still an issue of this not aligning properly when scaling and likely on mobile devices. We need another pass on  this.

I'd be fine pushing out this current version since it alleviates the problem of the particles demo for now.

Starkium committed:
Rework Style

Particles demo was causing issues for people, reworked this to be more compatible.

There's still an issue of this not aligning properly when scaling and likely on mobile devices. We need another pass on  this.

I'd be fine pushing out this current version since it alleviates the problem of the particles demo for now.

flare199 reviewed (approved):
Good as a temporary fix but we should optimize and clean it up later on

sshiiden commented:
You should use the SVG file for a clearer image.

**PNG**
![image](https://github.com/user-attachments/assets/d757fa9a-8aae-44c6-984e-8ef066fae0c7)
**SVG**
![image](https://github.com/user-attachments/assets/0f6be718-f9a0-4ff7-920c-9f42dbf789d2)

Starkium commented:
> You should use the SVG file for a clearer image.
> 
> **PNG** ![image](https://private-user-images.githubusercontent.com/95104877/376355620-d757fa9a-8aae-44c6-984e-8ef066fae0c7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg5Mzg2ODYsIm5iZiI6MTcyODkzODM4NiwicGF0aCI6Ii85NTEwNDg3Ny8zNzYzNTU2MjAtZDc1N2ZhOWEtOGFhZS00NGM2LTk4NGUtOGVmMDY2ZmFlMGM3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE0VDIwMzk0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgxNjVmMmI1NDU2Y2FlM2ViNTA1ZWU0OWJiZmI3MTljNTZkMTE4YzJhODc0ZTUwZTBhMDk4OTExYzhlMjg0YWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.PlKhaypYINQ29UdncgxKpLy-c-FstJTiDO6NjSYLXwE) **SVG** ![image](https://private-user-images.githubusercontent.com/95104877/376355658-0f6be718-f9a0-4ff7-920c-9f42dbf789d2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg5Mzg2ODYsIm5iZiI6MTcyODkzODM4NiwicGF0aCI6Ii85NTEwNDg3Ny8zNzYzNTU2NTgtMGY2YmU3MTgtZjlhMC00ZmY3LTkyMGMtOWY0MmRiZjc4OWQyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE0VDIwMzk0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBjN2MxMmYwOTFjZTZiZmE1N2ZhYzY5MWEyNTBmMjFhOWVkZDcxNTM5YTk4YjdjYTZmZWE1NWVlYWU4MTg5YjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.NFrwqQpXXxwUFFFUT6_7R8hmvIlrFRZEKklDCkEPqUQ)

I'm not a web dev so forgive me for potentially sounding dumb, but isn't svg more expensive?

sshiiden commented:
For simple images like logos, SVG files are less expensive.

Starkium committed:
Update style.css

remove particles reference

Starkium committed:
Merge branch 'master' into NewStyle

Starkium committed:
Update Logo and Point to New location

add white outline version of logo and point to new assets on github.
need to figure out how to point css to local directory

Starkium committed:
Update effects.css

colors and sizing of background tunnel

sshiiden commented on static/css/effects.css:
The png is used instead of the svg.

Starkium replied:
fixed, but sometimes the svg has an issue with aliasing when moving. I need to look into that later

Starkium committed:
Update effects.css

recolor again.
would like to get a layered background, stars maybe

someone suggested dust particles etc might be better

Starkium committed:
Update effects.css

use svg instead of png. It's got weird aliasing when using svg. Need to look into anti-aliasing. I think I saw a command in the particles demo earlier

Starkium committed:
Stash Example for Blanding Images

Gunna keep this here for when we get our own art
